platform/contracts/admin/src/contracts/protocol/mod.rs

```diff
@@ -21,6 +21,57 @@ pub struct ProtocolTemplate<T> {
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, JsonSchema)]
 #[serde(rename_all = "snake_case", deny_unknown_fields)]
 pub struct Protocol {
-    pub network: String,
+    pub network: Network,
+    pub dex: Dex,
     pub contracts: ProtocolTemplate<Addr>,
 }
+
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, JsonSchema)]
+#[serde(
+    rename_all = "PascalCase",
+    rename_all_fields = "snake_case",
+    deny_unknown_fields
+)]
+pub enum Network {
+    #[serde(alias = "NEUTRON")]
+    Neutron,
+    #[serde(alias = "OSMOSIS")]
```

Comment:
what's the idea of having aliases? They do allow supporting a second name, but only on deserialization.

Reply:
I put them specifically for this migration. Later on we'll remove them, but right now we need them in order to automatically convert them from `SCREAMING_SNAKE_CASE` to `PascalCase`.

Reply:
I deemed that testing well-known `serde` attributes is beyond our scope. I had put tests on the mixed casing as we haven't used that before and it is after all mixing them, that behaviors is not explicitly documented.

Reply:
OK, then put a TODO around to not forget their purpose and when supposed to be removed